tueplots/figsizes.py

```diff
@@ -326,6 +326,57 @@ def cvpr2024_full(
     )
 
 
+def probnum2025_half(
+    *,
+    nrows=1,
+    ncols=1,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """Single-column (half-width) figures for ProbNum 2025."""
+    figsize = _from_base_in(
+        base_width_in=3.333,
+        rel_width=1.0,
+        height_to_width_ratio=height_to_width_ratio,
+        nrows=nrows,
+        ncols=ncols,
+    )
```

Comment:
Do I remember correctly that you set the pagesizes via `pt`, not via inches? There is also a function `_from_base_pt` that could be used here. (Not that the result matters, only FYI)

Reply:
Ah, thanks for the pointer! I hadn't realised. Fixed in the commit below.